vismne/viz.py

```diff
@@ -330,7 +330,7 @@ def _add_colorbar(self, colormap, clim=None, size=(60, 4), height_max=100,
             rect = [-3 * w / 2, -h, w * 3, 5 * h]
             camera = scene.cameras.PanZoomCamera(rect=rect)
             # Create the subplot :
-            r = 1 if self._hemi in ['lh', 'rh'] else 2
+            r = 1 if self._hemi in ['lh', 'rh', 'both'] else 2
```

Comment:
works great! But do we need the `if-else`? In other words, is r=2 ever going to happen?

Reply:
Yes, if `hemi='split'`. Could be `r = 2 if self._hemi == 'split' else 1`. Could be more explicit

Reply:
Gotcha!